src/server_manager/model/cloud.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 The Outline Authors
```

Comment:
I think calling this "accounts.ts" is clearer, since this is kind of an accounts repository. Or even cloud_accounts.ts